modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -12,51 +12,32 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 * 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
 * 4 GB of storage
 
-== Minimum Software Requirements
+== Supported Software and Systems
 
 Verify that you use a supported version of Java and a supported operating system before you install Mule.
 
-If you use an application server or need to access a data store, verify that you use a supported application server or database.
+Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
 
-=== Supported Java Versions
+[%header,cols="15a,85a"]
+|===
+|Software |Version
+| JDK | JDK 1.8.0 (JDK 1.8.0_161 recommended), OpenJDK 8 (JDK 1.8.0_162 recommended), IBM JDK 1.8
+| OS | MacOS 10.11.x, HP-UX 11i V3, AIX 7.2, Windows 2016 Server (recommended), Windows 2012 R2 Server, Windows 10 (recommended), Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 16.04, or any other Linux distribution that runs a supported JDK and Linux kernel 3.13
+| Application Servers | Tomcat 7, Tomcat 8, Weblogic 12c, Wildfly 8, Wildfly 9, Websphere 8, Jetty 8, Jetty 9
+| Databases | Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
+| Browsers for working on Anypoint Platform | The latest versions of Firefox, Chrome, and Edge. Note that IE is being deprecated; minimum screen resolution of 1024x768
+|===
 
-* Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended)
-* IBM JDK 1.8
-* OpenJDK 8 (JDK 1.8.0_162 recommended)
+If you use the Microsoft Dynamics GP connector, install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
```

Comment:
Do we have this connector already available for Mule 4? Does this requirement still apply? If both answers are yes, do we still need to keep this on this page, or does it make sense to move it to the connectors documentation page?

Reply:
Yes, this connector is compatible with Mule 4 under the Connectors page https://docs.mulesoft.com/connectors/ms-dynamics/ms-dynamics-gp-connector#what-s-new-in-this-connector.
I'll remove this from Hardware and Software requirements and leave it under connectors. Though, I need to confirm if JCE is still required to be installed.